src/openrct2-ui/windows/EditorScenarioOptions.cpp

```diff
@@ -29,6 +30,8 @@
 #include <openrct2/world/Climate.h>
 #include <openrct2/world/Park.h>
 
+using namespace OpenRCT2;
```

Comment:
I think this is a bit overkill, since you'd only need to prepend the namespace once on line 436.

Reply:
This will actually help move all the things into the OpenRCT2 namespace at some point, there is no downside having it.

Reply:
Ok, your call.